Assets/_BForBoss/_Leaderboard/Scripts/Username/InputUsernameViewBehaviour.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace BForBoss
+{
+    /// <summary>
+    /// Placeholder view just to input the name
+    /// </summary>
+    public class InputUsernameViewBehaviour : MonoBehaviour
+    {
+        [SerializeField] private TMP_InputField _usernameField = null;
+        [SerializeField] protected TMP_Text _infoSettingsLabel = null;
+        [SerializeField] private Button _saveUsernameButton = null;
```

Comment:
[nitpick] should the field be called `_setUsernameButton` for consistency?

Reply:
brought up a good point, i'll also rename the class to be `SetUsernameViewBehaviour`